src/lib/解析資料.ts

```diff
@@ -15,7 +15,7 @@ export const m音韻編碼2反切 = new Map<string, string>();
       m音韻編碼2反切.set(編碼, 反切);
     }
 
-    const patternInner = /([^|])([^|]*)/gu;
+    const patternInner = /((?!\|).)((?:(?!\|).)*)/gu;
```

Comment:
This commit should fix issues of not being able to lookup characters outside Unicode BMP in production mode (i.e. on the qieyun-autoderiver website) due to a compilation bug of regular expression with a `u` flag.